{"name": "test_one", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 != 1", "trace": "attach_text_finalize = None\n\n    def test_one(attach_text_finalize):\n        with allure.step('Step 1'):\n            assert 10 == 10\n        with allure.step(\"Step 2\"):\n>           assert 1 != 1\nE           assert 1 != 1\n\ntest.py:48: AssertionError"}, "steps": [{"name": "Step 1", "status": "passed", "start": 1654938175870, "stop": 1654938175870}, {"name": "Step 2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 != 1\n", "trace": "  File \"/Users/eivankov/test_proj/test.py\", line 48, in test_one\n    assert 1 != 1\n"}, "start": 1654938175870, "stop": 1654938175870}], "start": 1654938175870, "stop": 1654938175870, "uuid": "853450e5-c5aa-4d58-b335-3fbdc4d64086", "historyId": "69ef32528934552b2c483953ffef0122", "testCaseId": "bcd8637debafe92c08009cc2f217014c", "fullName": "test#test_one", "labels": [{"name": "suite", "value": "test"}, {"name": "host", "value": "MacBook-Pro-Eugene.local"}, {"name": "thread", "value": "14276-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test"}]}